cluster.yml

```diff
@@ -2,6 +2,15 @@
 - name: Check ansible version
   import_playbook: ansible_version.yml
 
+- name: Add kube-master nodes to kube_control_plane
+  # This is for old inventory which contains kube-master instead of kube_control_plane
+  hosts: kube-master
+  gather_facts: false
+  tasks:
+    - name: add nodes to kube_control_plane group
+      group_by:
+        key: 'kube_control_plane'
```

Comment:
I'm wondering if in some cases we loose the `k8s-cluster` `groups_vars`.
Does a play with `hosts: kube_control_plane` get  `groups_vars` from `k8s-cluster` ?

Reply:
On the above case(kube-master exists instead of kube_control_plane), kube_control_plane has the same instances as kube-master and these instances are in k8s-cluster.
Can this be answer for your question?

Reply:
I'm just not sure how defaults work anymore, server mix the defaults from all groups they are in, or when you ask for `hosts: kube_control_plane`, you will get the server vars, the vars for `kube_control_plane` and not the vars from `k8s-cluster` because `kube_control_plane` is not in `k8s-cluster`

Reply:
OK, I am testing this PR locally to see the above question.

/hold

Reply:
confirmed this PR works with old inventory which contain kube-master instead of kube_control_plane.

/hold cancel

Reply:
Thanks for checking !